package.json

```diff
@@ -9,16 +9,26 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@mdx-js/loader": "^2.1.5",
+    "@mdx-js/react": "^2.1.5",
+    "@next/mdx": "^13.0.4",
+    "@types/mdx": "^2.0.3",
```

Comment:
I'm tempted to tell you to use TypeScript from day 0, but I didn't want to add too many complications (even though TypeScript is most of the time a help, rather than a complication, but you need to be yourself oriented to types)

Reply:
Thus, you're definitely not using this, at least for the time being :)

Reply:
Same, I have removed the unused packages.